CHANGELOG.md

```diff
@@ -72,6 +74,9 @@ https://github.com/googleapis/google-cloud-cpp/issues/8234.
   [IAM conditions][iam-conditions-link]. They did not work with IAM policies
```

Comment:
The PRs implementing this change appear multiple times below (#8667, #8652). It gets repetitive. Consider keeping this comment and removing the references to the PRs below.

Reply:
+1